### ISSUE TITLE
SSD trimming

### ISSUE THREAD

**Issue by andrimanna** [WEEE-Open/pesto#10]
To erase a SSD use TRIM with:
`# blkdiscard -f /dev/sdX`

**Comment by lvps** [comment]
See also secure erase: https://www.thomas-krenn.com/en/wiki/Perform_a_SSD_Secure_Erase

It's pretty cool and pretty secure

**Comment by il-palmi** [comment]
> Also: how to detect SSDs? Probably with lsblk, but this needs to be investigated

`lsblk -o NAME,ROTA`

**Comment by parmigggiana** [comment]
Is this supposed to substitute `badblocks` for SSDs? Is it ok to skip testing bad sectors?

**Comment by il-palmi** [comment]
Yes, we want to avoid writing extensively on SSDs to prevent unnecessary wear. Additionally, SSDs generally have built-in bad block management. While data on an HDD can be recovered by identifying patterns on the physical disk, this is much more difficult to achieve on an SSD due to the randomness of data distribution across its cells.
